### DISCUSSION TITLE
Unauthenticated visitors can search for content and can view tags

### DISCUSSION THREAD

**Discussion by phant0mski** [requarks/wiki#5640] [Help / Questions] [answered]
Hi there, 

Not sure if it's a bug or a feature. 
I just started to play around with wiki.js to see if it's a good fit for our needs. All the pages are made visible for Admin group. Guest group is excluded. 
Unauthenticated, not logged-in users, when browsing the home page cannot see any article but if they look up in the search bar using certain keywords they are able to view the articles. If they click on tags icon they see all the tags and by clicking on the tags can view related content. 

Am I doing something wrong? 

Thanks.

**Comment by NGPixel** [comment, marked as answer]
I'm guessing you mean the sidebar navigation when talking about pages guests can't see? Hiding something from the sidebar doesn't make the page itself private.

You need to set page rules (under Admin > Groups) to specify what the Guests group has access to. This will dictate what is shown in the sidebar, the tags view, the search results and whether the page can be seen if loaded directly.

**Reply by phant0mski** [reply]
Correct, I was referring to the sidebar items. Now I see how it works. I removed read permissions from Guest group and the content is not visible anymore. 
Thanks for your support. Much appreciated.

**Reply by stv33638** [reply]
I was having the same issue.  It was troublesome to test a simple and robust set of rules without example usages after searching.

I was able to hide and show based on Tags and the root path, but when I search the count of results still represent the 'private' pages that are not visible in the results.  This 'hint' of data and text on pages that guest doesn't have access to is concerning to me.  It appears that this count only appears if there is at least one hit that is readable to the guest.
